src/User.js

```diff
@@ -1,14 +1,47 @@
 class User {
-  constructor(user){
+  constructor(user) {
     this.id = user.id;
     this.name = user.name;
-    this.customer = user.customer;
     this.userBookings = [];
+    this.bookingCost = 0;
+
+  }
+  // Methods I think I'll need
+
+  retrieveRoomBookings(bookingData) {
+
+    bookingData.filter(booking => {
+      if (this.id === booking.userID) {
+        this.userBookings.push(booking);
+      }
+    });
+    // console.log(this.userBookings)
+    return this.userBookings;
+
   }
-}
 
+  calculateRoomCost(roomData) {
```

Comment:
Purely a performance suggestion, so not worth worrying about but an interesting topic...

Since you are putting the room cost into the booking data every time `calculateRoomCost` is called, you're asking javascript to do this multiple times. You could improve performance by instead setting the booking's `totalCost` in `retrieveRoomBookings` instead, which I assume you'll only call once.